.build/cspell-words.txt

```diff
@@ -1,16 +1,21 @@
 adfs
 Adsi
 amsi
-Antispam
+Andriod
 anob
+Antispam
+Appt
 authsspi
 autodiscover
 bincirc
 BSTR
 cach
 cachtokn
+Callog
+clientinfostring
```

Comment:
These need to be removed. 

CalLog should be the correct format for short handle of CalendarLog. This should pass. 

ClientInfoString is the correct format.